package.json

```diff
@@ -29,7 +29,7 @@
     "@google-cloud/common": "^0.20.3",
     "bindings": "^1.2.1",
     "console-log-level": "^1.4.0",
-    "delay": "^3.0.0",
+    "delay": "~3.0.0",
```

Comment:
Perhaps add a "TODO" here to fix this and link to an upstream bug?

Reply:
I'm not super happy with any method I've seen for commenting in package.json (several listed in [this](https://stackoverflow.com/questions/14221579/how-do-i-add-comments-to-package-json-for-npm-install) stackoverflow post).

Should I still add a comment?

Reply:
Huh, I definitely expected this to be simpler than that. Scratch that I guess.